server/utils/passport/ensureStudyPermission.js

```diff
@@ -0,0 +1,22 @@
+import UserModel from '../../models/UserModel'
+
+export default async function ensureStudyPermission(req, res, next) {
+  if (!req.isAuthenticated())
+    return res.status(401).json({ error: 'Unathorized' })
+  try {
```

Comment:
newline for readability.

```suggestion
    return res.status(401).json({ error: 'Unathorized' })

  try {
```

Reply:
[9ac5eed](https://github.com/AMP-SCZ/dpdash/pull/344/commits/9ac5eed8b55530c29fa5dc110bfc7de660805119)